lib/read.js

```diff
@@ -123,16 +129,23 @@ async function ProcessRead(msg, cfg) {
     const writer = new CsvWriter();
 
     debug('Sending GET request to url=%s', csvURL);
-    request.get(csvURL)
-        .on('response', async function onResponse(response) {
-            debug('Have got response status=%s headers=%j', response.statusCode, response.headers);
-            if (response.statusCode !== 200) {
-                await that.emit('error', 'Unexpected response code code=' + response.statusCode);
-                ended = true;
-                throw Error('Unexpected response code code=' + response.statusCode);
-            }
-        })
-        .pipe(parser).pipe(writer);
+    const ax = axios.create();
+    util.addRetryCountInterceptorToAxios(ax);
+    const response = await ax({
```

Comment:
Does Axios use IO stream to make PUT here?

Reply:
It is using streams. responseType: 'stream',